schema/scripts/schemas_consolidation/schemas_consolidation.py

```diff
@@ -690,16 +688,24 @@ def run_schemas_consolidation(
                 version_name = version["version_name"]
                 if version_name in version_names_list:
                     df_ref_v = df_ref[
-                        (df_ref["is_valid_v_" + version_name] == True)
-                        & (df_ref["is_downloaded"] == True)
+                        (df_ref["is_valid_v_" + version_name])
+                        & (df_ref["is_downloaded"])
                     ]
 
                     if len(df_ref_v) > 0:
                         # Get schema version parameters for ddup
                         version_dict = requests.get(version["schema_url"]).json()
-                        version_cols_list = [
+                        version_all_cols_list = [
                             field_dict["name"] for field_dict in version_dict["fields"]
                         ]
+                        version_required_cols_list = [
+                            field_dict["name"] for field_dict in version_dict["fields"]
+                            if (
+                                "constraints" in field_dict.keys()
+                                and "required" in field_dict["constraints"].keys()
+                                and field_dict["constraints"]["required"]
+                            )
```

Comment:
Could be simplified to this?
```suggestion
                            if field_dict.get("constraints", {}).get("required")
```